public/assets-sync.js

```diff
@@ -16,6 +16,7 @@
  * from the author.
  */
 
+/* eslint-disable */
```

Comment:
I didn't mean to commit this line, have to add it to make the file even slightly legible. Eslint *hates* something about it. We should probably try to fix that.